yt_dlp/extractor/nbc.py

```diff
@@ -142,18 +145,40 @@ def _real_extract(self, url):
         theplatform_url = smuggle_url(update_url_query(
             'http://link.theplatform.com/s/NnzsPC/media/guid/%s/%s' % (video_data.get('mpxAccountId') or '2410887629', video_id),
             query), {'force_smil_url': True})
+
+        # fallbacks
```

Comment:
Add a comment here why `or` is intentionally not used; so that someone else doesn't try to refactor it in the future

```suggestion
        # Empty string or 0 can be valid values for these. So the check must be `is None`
```